src/components/VoucherScanner/VoucherScanner.tsx

```diff
@@ -82,7 +82,7 @@ export const VoucherScanner: FunctionComponent<VoucherScanner> = ({
   const [showManualInput, setShowManualInput] = useState(false);
   useEffect(() => {
     const askForCameraPermission = async (): Promise<void> => {
-      const { status } = await Permissions.askAsync(Permissions.CAMERA);
+      const { status } = await Camera.requestPermissionsAsync();
```

Comment:
Can we use `granted` instead? (see [here](https://docs.expo.io/versions/v42.0.0/sdk/permissions/#permissionresponsegranted))

It seems more appropriate for determining if the user has granted sufficient permissions to access the camera.